docs/source/conf.py

```diff
@@ -10,9 +10,10 @@
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
 #
+import gemd
 import os
 import sys
-sys.path.insert(0, os.path.abspath('../../gemd'))
+sys.path.insert(0, os.path.abspath('../..'))
```

Comment:
Using apidoc causes that `reference/gemd.rst` file to be generated. By having it start outside of the gemd/ folder, you're causing its headline to be whatever folder contains `../..`. So I'd suggest leaving this as `../../gemd`.